### PR TITLE
fix: making side menu sticky

### LINKED ISSUE #3510: The sidebar is not sticky and the images on speakers page are all of different sizes.
**Is your feature request related to a problem? Please describe.**
The sidebar on the speakers and session page is not visible throughout the page. It disappears in the middle and suddenly appears at the end of the page. Also, the images of speakers are all in uneven sizes and some of them do not have captions and others have which makes it look odd.

**Describe the solution you'd like**

I would like to make the sidebar sticky at the side area and also change the image sizes to a suitable size along with a proper space for captions included.
**Describe alternatives you've considered**

**Additional context**

Here are the screenshots that can provide a better outlook of the problem:
issue2
issue3

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3510 
Making side bar sticky . 
working fine on computers : 
ezgif com-gif-maker (3)
And Mobile Device : 
Screenshot from 2019-11-11 15-23-09

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
@kushthedude @uds5501 please review this .

**Comment by kushthedude** [comment]
I think you didn't get the issue, We have to make it stick to the top irrespective of scrolls. Using `rails` the navbar is stick to the top and to navigate to other option I had to scroll up back.

**Comment by kushthedude** [comment]
See the second image on `issue description`, We have to make sure that doesn't happen.

**Comment by maze-runnar** [comment]
> See the second image on `issue description`, We have to make sure that doesn't happen.

@kushthedude  i have seen the issue description's second image . but once i refresh the page it disappeared .Making it sticky will be solve that problem (as user is not going to refresh the page every time:smile:) . As in profile section also the side menu is sticky and one have to scroll back to change the page(at https://eventyay.com/account/billing/payment-info). I think there should be no problem in scrolling back as they do in mobile phones .

**Comment by maze-runnar** [comment]
@kushthedude 
https://stackoverflow.com/questions/51102492/refreshing-semantic-ui-sticky-with-ember-does-not-work
can you take a look at this .

**Comment by kushthedude** [comment]
> > See the second image on `issue description`, We have to make sure that doesn't happen.
> 
> @kushthedude i have seen the issue description's second image . but once i refresh the page it disappeared .Making it sticky will be solve that problem (as user is not going to refresh the page every time😄) . As in profile section also the side menu is sticky and one have to scroll back to change the page(at https://eventyay.com/account/billing/payment-info). I think there should be no problem in scrolling back as they do in mobile phones .
> Screenshot from 2019-11-13 19-55-40

The sidebar should be stick to the top, irrespective of user scrolls or not.

**Comment by maze-runnar** [comment]
https://stackoverflow.com/questions/51102492/refreshing-semantic-ui-sticky-with-ember-does-not-work

**Comment by maze-runnar** [comment]
please take a look at this . this will make clear what i want to say .

**Comment by iamareebjamal** [comment]
@maze-runnar Use position sticky

**Comment by maze-runnar** [comment]
Ok

**Comment by maze-runnar** [comment]
> @maze-runnar Use position sticky

@iamareebjamal `position: sticky` is also not working. cause `ui menu` uses flex-box and `sticky` not works in this case.https://stackoverflow.com/questions/43707076/position-sticky-not-working-css-and-html/47878455

What i should do next?

**Comment by iamareebjamal** [comment]
See if there is overflow hidden in parent

**Comment by maze-runnar** [comment]
> See if there is overflow hidden in parent

ok

**Comment by maze-runnar** [comment]
@iamareebjamal actual problem is here : 
```
{{#ui-sticky context='#public-event-content' observeChanges=true}}
       {{public/side-menu event=model}}
 {{/ui-sticky}}
```
in {{ui-sticky}}.

I used 
```
div class="ui fluid" style="position: sticky; top:0px;">
      {{public/side-menu event=model}}
</div>
```

and it resolves the issue. should i remove `{{ui-sticky}}` ?

**Comment by iamareebjamal** [comment]
Yes, wherever used

File: app/templates/public.hbs [hunk 1]
```diff
@@ -35,9 +35,9 @@
             {{/if}}
           {{/if}}
         {{else}}
-          {{#ui-sticky context='#public-event-content' observeChanges=true}}
+          <div class="ui fluid" style="position:sticky; top:0px;">
```
Comment (iamareebjamal) [inline app/templates/public.hbs]: I don't think top 0px is right. Besides, it should be moved to CSS file

Comment (maze-runnar) [reply app/templates/public.hbs]: it's working on top:0px -  https://youtu.be/8cqPH3b4X58
and if i should change then what should be the value.

**Comment by iamareebjamal** [comment]
@maze-runnar top: 14px will look better

**Pull request merged**